pkg/apis/common/common.go

```diff
@@ -156,6 +156,20 @@ type SASLConfig struct {
 	PasswordSecret *corev1.SecretKeySelector `json:"passwordSecret,omitempty" protobuf:"bytes,3,opt,name=password"`
 }
 
+// SchemaRegistryConfig refers to configuration for a client
+type SchemaRegistryConfig struct {
+	// Schema Registry URL.
+	URL string `json:"url" protobuf:"bytes,1,opt,name=url"`
+	// +optional
+	// Api key is the authentication for registry
+	UserSecret *corev1.SecretKeySelector `json:"userSecret,omitempty" protobuf:"bytes,2,opt,name=user"`
```

Comment:
Can we keep the property names same in `json` and `protobuf`?

BTW, these can be replaced by using something like:

```golang
	// xxxxxx
	// +optional
	apicommon.BasicAuth `json:",inline" protobuf:"bytes,2,opt,name=basicAuth"`
```

Reply:
Okay! I'm using BasicAuth now.